src/features/page_view_timing/aggregate/index.js

```diff
@@ -54,16 +46,30 @@ export class Aggregate extends AggregateBase {
 
     const initialHarvestSeconds = getConfigurationValue(this.agentIdentifier, 'page_view_timing.initialHarvestSeconds') || 10
     const harvestTimeSeconds = getConfigurationValue(this.agentIdentifier, 'page_view_timing.harvestTimeSeconds') || 30
-    // send initial data sooner, then start regular
-    this.ee.on(`drain-${this.featureName}`, () => {
-      this.scheduler = new HarvestScheduler('events', {
-        onFinished: (...args) => this.onHarvestFinished(...args),
-        getPayload: (...args) => this.prepareHarvest(...args)
-      }, this)
-      this.scheduler.startTimer(harvestTimeSeconds, initialHarvestSeconds)
-    })
 
-    this.drain()
+    this.waitForFlags((['pvt'])).then(([pvtFlag]) => {
+      if (pvtFlag) {
+        firstPaint.subscribe(this.#handleVitalMetric)
+        firstContentfulPaint.subscribe(this.#handleVitalMetric)
+        firstInputDelay.subscribe(this.#handleVitalMetric)
+        largestContentfulPaint.subscribe(this.#handleVitalMetric)
+        interactionToNextPaint.subscribe(this.#handleVitalMetric)
+        timeToFirstByte.subscribe(({ entries }) => {
+          this.addTiming('load', Math.round(entries[0].loadEventEnd))
+        })
+
+        this.scheduler = new HarvestScheduler('events', {
```

Comment:
I think this is the only feature where we expose the harvest scheduler on the feature. I don't see that being used anywhere in this file. I wonder if this can just be made a local `const` like all the other features.